crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/CheckHealthCommand.java

```diff
@@ -0,0 +1,92 @@
+/*
+ Copyright 2013 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of lightblue.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.redhat.lightblue.rest.crud.cmd;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.codahale.metrics.health.HealthCheckRegistry;
+import com.redhat.lightblue.mediator.Mediator;
+import com.redhat.lightblue.rest.CallStatus;
+import com.redhat.lightblue.rest.crud.RestCrudConstants;
+import com.redhat.lightblue.rest.crud.health.CrudCheckRegistry;
+import com.redhat.lightblue.rest.crud.health.CrudCheckResponse;
+import com.redhat.lightblue.util.Error;
+import com.redhat.lightblue.util.metrics.RequestMetrics;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author dhaynes
+ */
+public class CheckHealthCommand extends AbstractRestCommand {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CheckHealthCommand.class);
+
+    private RequestMetrics metrics;
+    HealthCheckRegistry healthCheckRegistry;
+
+    public CheckHealthCommand(HealthCheckRegistry healthCheckRegistry, RequestMetrics metrics) {
+        this(null, healthCheckRegistry, metrics);
+    }
+
+    public CheckHealthCommand(Mediator mediator, HealthCheckRegistry healthCheckRegistry, RequestMetrics metrics) {
+        super(mediator);
+        this.healthCheckRegistry = healthCheckRegistry;
+        this.metrics = metrics;
+    }
+
+    @Override
+    public CallStatus run() {
+
+        RequestMetrics.Context context = metrics.startHealthRequest();
```

Comment:
Yeah I keep forgetting that those checks are running async...I'll leave out the duration addition for now, but we can add later if we find ourselves looking for it.

Not sure if I understand what you mean by "adding real JMX metric tracking", don't I get that with the metrics instance that is set locally here (passed in from AbstractCrudResoure)?